<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentProjectView</class>
 <widget class="QDialog" name="studentProjectView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student Projects</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>351</width>
     <height>341</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="registerButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLabel" name="projectName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>25</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Project Name</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="projectDescription">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="manageProjectButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="projectsList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>161</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
